collect_app/src/main/res/layout/project_settings_dialog_layout.xml

```diff
@@ -11,13 +11,6 @@
         android:orientation="vertical"
         app:layout_constraintGuide_begin="@dimen/margin_standard" />
 
-    <androidx.constraintlayout.widget.Guideline
```

Comment:
I liked the first version more but that's not the most important thing here. The thing is that we shouldn't rather implement changes like this one - difficult to spot and not directly related to the subject of this pr. And also centering itself is not that important, more important was to have the whole area clickable what you also changed and now you have to click directly on a button to trigger any action.

Reply:
That's fair. I should really have called this out. I ended up making this change as I ran into some weird click propagation problems in the commit before (where I had to move the IDs around) and I was pretty sure I could simplify/flatten the view, but you're right that I didn't keep some behaviour intact. I'll see what I can do about increasing the click area for the buttons. 

It's also worth pointing out that it's best to avoid wrapping buttons in another clickable view, as it could cause problems for accessibility (unless we do extra work to make sure content descriptions are set up correctly). Basically, if the tests can't access the button with text, then screen readers will be confused as well. We might just need a custom view for this kind of thing. Again, this is all things I should have called out as part of the PR description/comments, so sorry about that!